app/models/person.rb

```diff
@@ -103,6 +103,8 @@ class Person < ActiveRecord::Base
   has_many :authored_comments, :class_name => "Comment", :foreign_key => "author_id", :dependent => :destroy
   has_many :community_memberships, :dependent => :destroy
   has_many :communities, -> { where("community_memberships.status = 'accepted'") }, :through => :community_memberships
+  has_one  :community_membership, :dependent => :destroy
+  has_one  :community, -> { where("community_memberships.status= 'accepted'") }, through: :community_membership
```

Comment:
How do you feel about this? Should we follow the same path as the existing implementation (community through membership where status is accepted), or should we just simply say `has_one :community` and use the `person.community_id` as the foreign key? I think that the latter would be a bit simpler, but on the other hand that's then changing the logic (status doesn't need to be accepted). If we decide to change the logic, then that needs to be taken into account when changing the deprecated `communities` to `community`.